nlmod/mdims/mgrid.py

```diff
@@ -102,200 +101,74 @@ def get_cell2d_from_model_ds(ds):
     return cell2d
 
 
-def get_xy_mid_structured(extent, delr, delc, descending_y=True):
-    """Calculates the x and y coordinates of the cell centers of a structured
-    grid.
-
-    Parameters
-    ----------
-    extent : list, tuple or np.array
-        extent (xmin, xmax, ymin, ymax)
-    delr : int or float,
-        cell size along rows, equal to dx
-    delc : int or float,
-        cell size along columns, equal to dy
-    descending_y : bool, optional
-        if True the resulting ymid array is in descending order. This is the
-        default for MODFLOW models. default is True.
-
-    Returns
-    -------
-    x : np.array
-        x-coordinates of the cell centers shape(ncol)
-    y : np.array
-        y-coordinates of the cell centers shape(nrow)
+def refine(ds, model_ws=None, refinement_features=None, exe_name=None,
```

Comment:
refine is a bit vague, should we specify a bit more: refine_gridgen, refine_lgr, refine_quadtree?

Reply:
Hmm, refine_gridgen or refine_quadtree is ok by me as well. But I think we should make a choice which refinement techniques we would like to support, also keeping in mind the resampling techniques we would like to support in the future. I think we should only support quadtree grids, and therefore would simply name this method 'refine'. It is allready in the grid-module, which clarifies what we are refining. But let's discuss this at the next meeting.

Reply:
I see what you're saying, I guess i just like explicitly knowing I'm going to be using gridgen or something. But I can also live with the succinctness of "refine". It should be very clear that it does not work on triangles or voronoi grids, but maybe that is already logical.